docs/upgrading/from-v8.md

```diff
@@ -1,4 +1,8 @@
-# Migrating from v8 to v9
```

Comment:
We should keep the `UPGRADING.md` and provide a link to the new location as it's referenced in GitHub issues quite a few times.

On top of that I think that this page should be a single page not scoped to a version, so it'd be `/docs/upgrading` instead of `/upgrading/from-v8` which will be outdated when never versions are released.

Reply:
@timneutkens ok! I actually searched through github issues and there were only 7, but I guess I didn't catch them all:

<img width="683" alt="Screen Shot 2020-02-28 at 10 04 52 AM" src="https://user-images.githubusercontent.com/992008/75573506-d6a06080-5a11-11ea-9f25-0fc5353371eb.png">

I was going to make separate pages for newer versions like:

- `/upgrading/from-v8` (handles upgrading from v8 to v9.0.x)
- `/upgrading/from-v9.2` (handles upgrading from v9 to v9.3)

But you're right, it can be in one page. So I updated it!